trk/src/Screens/GymDaily/index.js

```diff
@@ -1,8 +1,219 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useContext } from 'react';
 import { Text, View, ScrollView, SafeAreaView, StyleSheet } from "react-native";
+import { AuthContext } from '../../Utils/AuthContext';
+import ClimbsApi from '../../api/ClimbsApi';
+import TapsApi from '../../api/TapsApi';
+import CommentsApi from '../../api/CommentsApi';
+
 
 const GymDaily = () => {
 
+  const [yourClimbs, setYourClimbs] = useState([]);
+  const [yourComments, setYourComments] = useState([]);
+  const [totalClimbs, setTotalClimbs] = useState('');
+  const [asents, setAscents] = useState([]);
+
+  const [totalAscents, setTotalAscents] = useState('');
+  const [latestAscents, setLatestAscents] = useState('');
+  const [mostCompleted, setMostCompleted] = useState('');
+  const [leastCompleted, setLeastCompleted] = useState('');
+  const [peakTime, setPeakTime] = useState('');
+  const [highestRated, setHighestRated] = useState({
+    name: '',
+    grade: '',
+    type: '',
+    climb: '',
+  });
+  const [latestFeedback, setLatestFeedback] = useState({
+    explanation: '',
+    climb: '',
+    rating: '',
+    climbName: '',
+  });
+
+  const { currentUser } = useContext(AuthContext);
+  const { getClimbsBySomeField } = ClimbsApi();
+  const { getTapsBySomeField } = TapsApi();
+  const { getCommentsBySomeField } = CommentsApi();
+  const { getClimb } = ClimbsApi();
+
+  const userId = currentUser.uid;
+
+
+  useEffect(() => {
+    const fetchClimbs = async () => {
+      try {
+        const querySnapshot = await getClimbsBySomeField('setter', userId);
+        if (querySnapshot && querySnapshot.docs) {
+          const climbs = querySnapshot.docs.map(doc => {
+            return {
+              id: doc.id,
+              ...doc.data()
+            };
+          }).filter(climb => !climb.archived); // Filter out archived climbs
+
```

Comment:
made sure to remove archived climbs from the list of you set climbs so they are not included in any calculation